Broken links in Docusaurus preventing website from building

While reviewing #2942 , I hit an error that prevents Docusaurus from building the website

How to reproduce:
- `cd src/docs`
- `pnpm build`

Output:
```
PS C:\Users\lecin\Documents\repo\telescope\src\docs> pnpm build

> telescope-docs@0.0.1 build C:\Users\lecin\Documents\repo\telescope\src\docs
> docusaurus build

[INFO] [en] Creating an optimized production build...

✔ Client
  

✔ Server
  Compiled successfully in 5.96s


✔ Client
  

● Server █████████████████████████ cache (99%) shutdown IdleFileCachePlugin
 stored

[ERROR] Unable to build website for locale en.
[ERROR] Error: Docusaurus found broken links!

Please check the pages of your site in the list below, and make sure you don't reference any path that does
Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the b

Exhaustive list of all broken links found:

- On source page path = /docs/api-services/api:
   -> linking to docker/docker-compose.yml (resolved as: /docs/api-services/docker/docker-compose.yml)
   -> linking to docker/development.yml (resolved as: /docs/api-services/docker/development.yml)
   -> linking to docker/production.yml (resolved as: /docs/api-services/docker/production.yml)
   -> linking to config/env.development (resolved as: /docs/api-services/config/env.development)
   -> linking to config/env.staging (resolved as: /docs/api-services/config/env.staging)
   -> linking to config/env.production (resolved as: /docs/api-services/config/env.production)

- On source page path = /docs/getting-started/environment-setup:
   -> linking to ../config/env.development (resolved as: /docs/config/env.development)
   -> linking to ../config/env.production (resolved as: /docs/config/env.production)
   -> linking to ../config/env.staging (resolved as: /docs/config/env.staging)

- On source page path = /docs/tools-and-technologies/login:
   -> linking to kristophjunge/test-saml-idp (resolved as: /docs/tools-and-technologies/kristophjunge/test-
   -> linking to ../simplesamlphp-users.php (resolved as: /docs/simplesamlphp-users.php)

- On source page path = /docs/tools-and-technologies/pino:
   -> linking to ../src/backend/utils/logger.js (resolved as: /docs/src/backend/utils/logger.js)

- On source page path = /docs/tools-and-technologies/renovate:
   -> linking to ../.github/renovate.yml (resolved as: /docs/.github/renovate.yml)

    at reportMessage (C:\Users\lecin\Documents\repo\telescope\node_modules\.pnpm\@docusaurus+utils@2.0.0-be
    at handleBrokenLinks (C:\Users\lecin\Documents\repo\telescope\node_modules\.pnpm\@docusaurus+core@2.0.0    at async mapAsyncSequential (C:\Users\lecin\Documents\repo\telescope\node_modules\.pnpm\@docusaurus+utils@2.0.0-beta.15_82301ea35a8edadc6a7972d57c866c58\node_modules\@docusaurus\utils\lib\index.js:200:24)
    at async build (C:\Users\lecin\Documents\repo\telescope\node_modules\.pnpm\@docusaurus+core@2.0.0-beta.15_182b7aa9b17480f4d87870e9c5f139c0\node_modules\@docusaurus\core\lib\commands\build.js:71:25)
 ELIFECYCLE  Command failed with exit code 1.
```

--- a/src/docs/docs/api-services/api.md
+++ b/src/docs/docs/api-services/api.md
@@ -4,19 +4,19 @@
 
 There are a number of [docker-compose](https://docs.docker.com/compose/) files available for running Telescope's services, including:
 
-- [docker/docker-compose.yml](docker/docker-compose.yml) - the base docker-compose stack
+- [docker/docker-compose.yml](../../../../docker/docker-compose.yml) - the base docker-compose stack
   definition of our microservices, dependent services, and our api gateway, [Traefik](https://traefik.io). This is meant to be used in development only.
-- [docker/development.yml](docker/development.yml) - overrides and extensions to the base docker-compose
+- [docker/development.yml](../../../../docker/development.yml) - overrides and extensions to the base docker-compose
   stack, with extra settings and services necessary for running in development or CI.
-- [docker/production.yml](docker/production.yml) - overrides and extensions to the base docker-compose
+- [docker/production.yml](../../../../docker/production.yml) - overrides and extensions to the base docker-compose
   stack, with extra settings and services necessary for running in staging and production.
 
 In conjunction with these, there are also multiple environment files, including:
 
-- [config/env.development](config/env.development) - environment variables for local development. This
+- [config/env.development](../../../../config/env.development) - environment variables for local development. This
   is likely what you need to adjust or pay attention to if you're developing Telescope.
-- [config/env.staging](config/env.staging) - environment variables for our staging server.
-- [config/env.production](config/env.production) - environment variables for our production server.
+- [config/env.staging](../../../../config/env.staging) - environment variables for our staging server.
+- [config/env.production](../../../../config/env.production) - environment variables for our production server.
 
 The env files are configured to specify which variables and docker-compose files are needed to be run.
 For most developers, doing the following will work:

--- a/src/docs/docs/getting-started/environment-setup.md
+++ b/src/docs/docs/getting-started/environment-setup.md
@@ -242,9 +242,9 @@ Docker builds Telescope's dependencies at launch and keeps them on disk. In some
 
 ### Start Telescope
 
-There are different ways to run the application. By default, [env.development](../config/env.development) will be used. Please read the use cases below to find out what configuration you need to make for different scenarios.
+There are different ways to run the application. By default, [env.development](../../../../config/env.development) will be used. Please read the use cases below to find out what configuration you need to make for different scenarios.
 
-There are also [env.production](../config/env.production) and [env.staging](../config/env.staging) to choose based on developer's need. For example, if you want to use staging, you can do `cp ./config/env.staging ./.env` on Linux/macOS, or `copy config/env.staging .env` on Windows.
+There are also [env.production](../../../../config/env.production) and [env.staging](../../../../config/env.staging) to choose based on developer's need. For example, if you want to use staging, you can do `cp ./config/env.staging ./.env` on Linux/macOS, or `copy config/env.staging .env` on Windows.
 
 Here are instructions for different scenarios:
 

--- a/src/docs/docs/tools-and-technologies/login.md
+++ b/src/docs/docs/tools-and-technologies/login.md
@@ -46,7 +46,7 @@ users) or `protectAdmin` (admin users) middleware from `src/backend/web/authenti
 
 ## Running an SSO Identity Provider
 
-We use a test SAML SSO provider via a [docker image](kristophjunge/test-saml-idp).
+We use a test SAML SSO provider via a [docker image](https://hub.docker.com/r/kristophjunge/test-saml-idp/).
 You can [read more about how it works here](https://medium.com/disney-streaming/setup-a-single-sign-on-saml-test-environment-with-docker-and-nodejs-c53fc1a984c9).
 
 To get this to run locally, you need to update your `.env`. Copy `env.example` if you don't have one, and the default values should work.
@@ -59,7 +59,7 @@ If running Elasticsearch or Redis natively, please avoid using `docker-compose u
 
 :::
 
-5. The test SSO server uses the following fake user accounts defined in [simplesamlphp-users.php](../simplesamlphp-users.php):
+5. The test SSO server uses the following fake user accounts defined in [simplesamlphp-users.php](../../../../config/simplesamlphp-users.php):
 
 | Username    | Password  | Name            | Email                      |
 | ----------- | --------- | --------------- | -------------------------- |

--- a/src/docs/docs/tools-and-technologies/pino.md
+++ b/src/docs/docs/tools-and-technologies/pino.md
@@ -4,7 +4,7 @@ sidebar_position: 5
 
 # Logging Support Using Pino
 
-This project uses [Pino](http://getpino.io/#/) to provide support for logging in Production as well as development environments. The [logger.js](../src/backend/utils/logger.js) module exports a logger instance that can be used in other modules to implement logging for important events.
+This project uses [Pino](http://getpino.io/#/) to provide support for logging in Production as well as development environments. The [logger.js](../../../backend/utils/logger.js) module exports a logger instance that can be used in other modules to implement logging for important events.
 
 ## How to use the logger
 

--- a/src/docs/docs/tools-and-technologies/renovate.md
+++ b/src/docs/docs/tools-and-technologies/renovate.md
@@ -4,7 +4,7 @@ sidebar_position: 8
 
 # Renovate and Telescope
 
-Telescope uses [Renovate](https://docs.renovatebot.com/) to check outdated dependencies and generate PRs to update dependencies. By default, Renovate checks all `package.json`, `dockerfile`, `docker-compose` and `github-actions` files for outdated dependencies. For more information such as PR schedule or limitation, please check [renovate.yml](../.github/renovate.yml)
+Telescope uses [Renovate](https://docs.renovatebot.com/) to check outdated dependencies and generate PRs to update dependencies. By default, Renovate checks all `package.json`, `dockerfile`, `docker-compose` and `github-actions` files for outdated dependencies. For more information such as PR schedule or limitation, please check [renovate.json](../../../../.github/renovate.json)
 
 ## How to handle Renovate PRs
 

